kotsadm/web/src/components/apps/AppConfig.jsx

```diff
@@ -271,6 +272,50 @@ class AppConfig extends Component {
     this.setState({ showNextStepModal: false });
   }
 
+  renderConfigInfo = (app) => {
+    const { match } = this.props;
+    const sequence = parseInt(match.params.sequence);
+    const currentSequence = app?.downstreams[0]?.currentVersion?.parentSequence;
+    const pendingVersions = app?.downstreams[0]?.pendingVersions;
+
+    if (currentSequence > sequence) {
+      return (
+        <div className="ConfigInfo older u-marginBottom--20">
+          <p className="flex alignItems--center u-marginRight--5"> <span className="icon info-warning-icon flex u-marginRight--5" /> This config is {currentSequence - sequence} version{currentSequence - sequence === 1 ? "" : "s"} older than the currently deployed config. </p>
+          <Link to={`/app/${app?.slug}/config/${currentSequence}`} className="replicated-link"> Edit the currently deployed config </Link>
+        </div>
+      )
+    } else if (currentSequence < sequence) {
+      return (
+        <div className="ConfigInfo newer u-marginBottom--20">
+          <p className="flex alignItems--center u-marginRight--5"> <span className="icon info-icon flex u-marginRight--5" /> This config is {sequence - currentSequence} version{sequence - currentSequence === 1 ? "" : "s"} newer than the currently deployed config. </p>
+          <Link to={`/app/${app?.slug}/config/${currentSequence}`} className="replicated-link"> Edit the currently deployed config </Link>
+        </div>)
+    } else if (size(pendingVersions) > 0 && (currentSequence === sequence)) {
+      return (
+        <div className="ConfigInfo current u-marginBottom--20">
+          <p className="flex alignItems--center u-marginRight--5"> <span className="icon info-icon-green flex u-marginRight--5" /> This is the currently deployed config. There {size(pendingVersions) === 1 ? "is" : "are"} {size(pendingVersions)} newer version{size(pendingVersions) === 1 ? "" : "s"} since this one. </p>
+          <Link to={`/app/${app?.slug}/config/${pendingVersions[0].parentSequence}`} className="replicated-link"> Edit the latest config </Link>
+        </div>
+      )
+    } else {
+      return null;
+    }
+  }
+
+  checkIsCurrentOrPastVersion = (app) => {
+    const { match } = this.props;
+    const sequence = parseInt(match.params.sequence);
+    const currentSequence = app?.downstreams[0]?.currentVersion?.parentSequence;
+
+    if (sequence <= currentSequence) {
```

Comment:
will this work for the initial config screen when installing an app?

Reply:
yes. because at that point we don't have anything in params. url looks like: `/app/:appSlug/config`.
All these changes are visible when viewing config for specific sequence. `/app/:appSlug/config/:sequence"